src/Raven.Server/Documents/Patch/AdminJsConsole.cs

```diff
@@ -47,6 +47,9 @@ public DynamicJsonValue ApplyServerScript(AdminJsScript script)
         {
             var jintEngine = GetEngine(script, ServerExeceutionStr);
 
+            jintEngine.Global.Delete("SecedeFromCluster", false);
+            jintEngine.SetValue("SecedeFromCluster", (Action)(() => _server.ServerStore.SecedeFromCluster()));
```

Comment:
Do we actually need this? We already have the server exposed here, we can just call `server.ServerStore.SecedeFromCluster()`, no?